Jenkinsfile

```diff
@@ -162,12 +162,19 @@ pipeline {
 
     stages {
         stage('trigger_sles_build') {
-          when {
+            when {
                 expression { return params.TRIGGER_SLES_BUILD }
-          }
-          steps {
-            build job: 'scf-sles-trigger', wait: false, parameters: [string(name: 'JOB_NAME', value: env.JOB_NAME)]
-          }
+            }
+            steps {
+                build(
+                    job: 'scf-sles-trigger',
+                    wait: false,
+                    parameters: [
+                        string(name: 'JOB_NAME', value: env.JOB_NAME),
+                        string(name: 'BRANCH_NAME', value: params.BRANCH_NAME),
```

Comment:
I think we have to pass `env.BRANCH_NAME` if `params.BRANCH_NAME` is not set in order to cover all cases, no?

Reply:
You're right; fixed in 1de8875bfcae66579063360a7be5a72dd913829f

Though now that I think about it, basing things on `scm.GIT_BRANCH` would have been easier, but needs more testing  (because I am not sure that actually works).